### ISSUE TITLE
Documentation: https://samltest.id/ doesn't exist anymore

### ISSUE THREAD

**Issue by orazionelson** [IQSS/dataverse#10872]
Hello,
the domain 

http://samltest.io/ 

is no longer active. 
The manual recommends testing Shibboleth connections using the following link: 

https://guides.dataverse.org/en/latest/installation/shibboleth.html. 

You can consider alternatives like: https://mocksaml.com/

**Comment by orazionelson** [comment]
In the 6.8 documentation, the site https://samltest.id/ still appears, which no longer exists.

**Comment by pdurbin** [comment]
@orazionelson yeah, you'd right. Have you (or anyone reading this) tried https://mocksaml.com ? Does it work?
